app/helpers/proctoring/tokens_helper.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+require 'jwt'
+require 'securerandom'
+
+module Proctoring
+  module TokensHelper
+    def generate_authentication_token(params)
+      generate_app_token(
+        params[:room_id],
+        params[:user_id],
+        params[:role],
+        Proctoring.hundred_ms_app_access_key,
+        Proctoring.hundred_ms_app_secret
+      )
+    end
+
+    def generate_management_token
+      management_token(
+        Proctoring.hundred_ms_app_access_key,
+        Proctoring.hundred_ms_app_secret
+      )
+    end
+
+    def self.encode_authentication_token
+      now = Time.now
+      exp = now + 10.minutes
+      payload = {
+        app_name: Proctoring.app_name,
+        type: 'application',
+        jti: SecureRandom.uuid,
+        version: 2,
+        iat: now.to_i,
+        nbf: now.to_i,
+        exp: exp.to_i
+      }
+      JWT.encode payload, Proctoring.hundred_ms_auth_secret, 'HS256'
+    end
+
+    private
+
+    def generate_app_token(room_id, user_id, role, app_access_key, app_secret)
+      now = Time.now
+      exp = now + 1.day
```

Comment:
`now`? can we rename it? to current_time or something else? also `expiry_time` 

applies everywhere.